.github/workflows/main.yml

```diff
@@ -6,14 +6,14 @@ name: CI
 # events but only for the master branch
 on:
   push:
-    branches: [ master ]
+    branches: [ master, maint-1.3 ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, maint-1.3 ]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
-  build:
+  ubuntu:
```

Comment:
Would it be possible to specify specific versions of Ubuntu?

Reply:
It is possible, but maybe it would be great to follow the edge in our CI?

Reply:
what about using matrix to run multiple jobs: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

you can run the action on multiple versions at the same time

Reply:
I'm not sure how hardly we could abuse GH builders, but okay. What do we want? ubuntu-latest + ubuntu 18.04 + ubuntu 20.04; macos-latest?

Reply:
Options: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners

Reply:
Hold on a second. I forgot Windows :). I'll add it

Reply:
Windows are already covered by AppVeyor and old Ubuntu is already covered by Travis.